doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -30,42 +100,18 @@ Kubernetes itself is a “system for automating deployment, scaling, and managem
 - monitoring and logging options
 - etc.
 
-Therefore, when considering software validation, and in NFV scenarios the compliance, verification and certification complexities, more thought is required around how flexible the "Platform" should be, with regards to these add-ons.  i.e. does a single blueprint for a "Kubernetes Platform for NFV" include specific versions of specific projects for each add-on, meaning that if you change from, e.g. Envoy to Fluentd, a new blueprint is created to certify against.
-
-There are a number of different approaches that could be adopted:
-1. Software vendor brings Kubernetes
-    - This option places the operator purely as the infrastructure provider, and the software vendor uses the VIM, or IaaS APIs, to manage the lifecycle of the infrastructure required to create, update and delete Kubernetes clusters ***and*** the software being deployed into Kubernetes
-2. Operator brings Kubernetes
-    - This option places the operator as not only the infrastructure provider, but also a "platform-as-a-service" provider (for Kubernetes Platform). The operator may already be providing platform-as-a-service offerings such as database-as-a-service, or other underlying "enabler services" for the software such as DNS, NTP, mail relay, etc.
-    - It is not necessarily the case that the vendor that the operator chooses for Kubernetes is the same vendor that provides the NFVI and/or VIM for that operator. That is a procurement decision for each operator.
-
-The main pros and cons to these approaches can be seen in the table below. This is broadly similar to the discussion had in the early days of virtualisation about whether or not to share NFVI platforms.
-
-| Option | Pros | Cons | Notes |
-| --- | --- | --- | --- |
-| 1. Software vendor brings Kubernetes | Potentially quicker validation/certification | Potential for silo deployments, operational complexity | ... |
-| 2. Operator brings Kubernetes | Maximum efficiency of infrastructure | Good for those operators building capabilities | ... |
-
-Suggested recommendation:
-- Option 2, operator brings Kubernetes to meet model for other "as-a-Service" components that are required such as DNS, NTP, etc.
-
-Figure 1-1 below shows this at a high level.
+Figure 1-1 below shows how Kubernetes, CaaS, and the CaaS Manager might fit within the current ETSI architecture.  In terms of this Reference Architecture, we should focus on the requirements of the new interfaces and the capabilities within the CaaS Manager and the Kubernetes clusters.
 
 <p align="center"><img src="../figures/ch01_k8s_arch.png" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="100%"/></p>
 <p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
 
-### Other thoughts
-- Kubernetes / CaaS should be recognised in ETSI NFV / MANO
```

Comment:
It is recognized. IFA029 says:
Container Infrastructure Service: Service that provides runtime environment for one or more container virtualization technologies.
NOTE 4:              Container Infrastructure Service can run on top of a bare metal or hypervisor-based virtualization

Container Infrastructure Service Management: a function that manages one or more Container Infrastructure Services.
NOTE 5:              It provides mechanisms for lifecycle management of the containers, which are hosting application components as services or functions.

Container Infrastructure Service Instance: an instance providing runtime execution environment for container.
EXAMPLE 2:     In Kubernetes® this is a node, which consists of three components: container runtime, kubelet and kube-proxy.

Reply:
Thank you - will update to reflect this.

Reply:
Minor correction: ETSI GR NFV-IFA 029 V0.14.0 (2019-03) no longer numbers the notes or the example.